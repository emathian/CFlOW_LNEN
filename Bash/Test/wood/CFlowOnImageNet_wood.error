Loading pytorch-gpu/py3/1.9.0
  Loading requirement: cuda/11.2 nccl/2.9.6-1-cuda cudnn/8.1.1.33-cuda gcc/8.4.1
    openmpi/4.1.1-cuda intel-mkl/2020.4 magma/2.5.4-cuda sox/14.4.2
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 1024 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfs7kw/linkhome/rech/genkmw01/ueu39kt/cflow-ad/FrEIA/modules/all_in_one_block.py:119: UserWarning: Soft permutation will take a very long time to initialize with 2048 feature channels. Consider using hard permutation instead.
  warnings.warn(("Soft permutation will take a very long time to initialize "
/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torchvision-0.10.0a0+300a8a4-py3.9-linux-x86_64.egg/torchvision/transforms/transforms.py:280: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
/gpfslocalsup/pub/anaconda-py3/2021.05/envs/pytorch-gpu-1.9.0+py3.9/lib/python3.9/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  ../c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
