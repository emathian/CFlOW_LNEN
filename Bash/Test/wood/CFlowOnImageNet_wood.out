LR schedule: [12, 18, 22]
Number of pool layers = 3
CNF coder: 512
CNF coder: 1024
CNF coder: 2048
train/test loader length 247 79
train/test loader batches 7 3
Loading weights from /gpfsscratch/rech/ohv/ueu39kt/CFLOW/weights/wood/mvtec_wide_resnet50_2_freia-cflow_pl3_cb8_inp512_run0_wood_2022-08-04-15:49:54.pt

Compute loss and scores on test set:
Epoch: 0 	 test_loss: 3629071.6570 and 5.70 fps
Heights/Widths [64, 32, 16] [64, 32, 16]
super_mask  (79, 512, 512)
DET_AUROC: 	 last: 98.16 	 max: 98.16 	 epoch_max: 0
SEG_AUROC: 	 last: 96.63 	 max: 96.63 	 epoch_max: 0
